models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: databricks-dolly-v2-12b
+path: ./
+properties:
+  SHA: 3f452b6e5a89b0e6c828c9bba2642bc577086eae
+  datasets: databricks/databricks-dolly-15k
+  inference-min-sku-spec: 4|1|28|176
```

Comment:
@haritha-bendapudi, @praveenmathamsetty Could you please confirm these?

Reply:
16|0|56|112